datasets/simulated.py

```diff
@@ -1,5 +1,6 @@
 from benchopt import BaseDataset
 from benchopt import safe_import_context
+from benchopt.datasets import make_correlated_data
```

Comment:
move this import to the safe_import block below so that it speeds up import and eases autocompletion